contrib/build_rpm.sh

```diff
@@ -53,13 +60,13 @@ export extra_arg="--without doc --without debug"
 echo ${PKGS[*]}
 sudo $pkg_manager install -y ${PKGS[*]}
 
+# clean up src.rpm as it's been built
```

Comment:
I would've preferred changing the comment, since it's now misleading, but it's not worth rerunning CI. Maybe next time take a quick look for context when you cut and paste.

LGTM. Thank you for your work on this.